data/wp/wp-content/plugins/epfl/lib/admin-controller.php

```diff
@@ -550,9 +552,8 @@ function _filter_manage_custom_column ($column, $post_id)
 
     function _filter_manage_sortable_columns ($columns)
     {
-        if ($this->sort_opts) {
-            $columns[$this->slug] = $this->slug;
-        }
+        if (! property_exists($this, 'sort_opts')) return;
```

Comment:
Si tu veux être aligné avec le code précédent, faudrait faire un `return $columns` au lieu d'un `return` non ?

Ou alors juste remplacer le code original `if ($this->sort_opts) {` par `if (property_exists($this, 'sort_opts')){`

Reply:
Fait avec `isset`, selon discussion ce midi et pour uniformité

Reply:
Pas sûr de comprendre pourquoi tu ne retournes rien avec la fonction si "sort_opts" n'existe pas. Si je compare avec le code original, il n'y aura pas le même comportement, c'est voulu ?

Reply:
Wé, quand j'oublie pas de `git push` c'est mieux...

Reply:
😆